packages/blueprints-integration/src/api.ts

```diff
@@ -146,7 +146,7 @@ export interface ShowStyleBlueprintManifest extends BlueprintManifestBase {
 		context: ISyncIngestUpdateToPartInstanceContext,
 		existingPartInstance: BlueprintSyncIngestPartInstance,
 		newData: BlueprintSyncIngestNewData,
-		playoutStatus: 'current' | 'next'
+		playoutStatus: 'previous' | 'current' | 'next'
```

Comment:
I think it would be good to add a note to the method docs to state when this will be called for 'previous'

Reply:
Added a note on when `playStatus: previous` and which PartInstance is received.